RefactoringPracticeI/src/test/java/org/assuresoft/inventory/GameInventoryTest.java

```diff
@@ -0,0 +1,45 @@
+package org.assuresoft.inventory;
+
+import org.assuresoft.middlewares.AgedBrieUpdateRule;
+import org.assuresoft.middlewares.UpdateRules;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class GameInventoryTest {
+  private final String ITEM_NAME = "foo";
+  private List<Item> items;
+  private GameInventory app;
+
+  @BeforeEach
```

Comment:
Subjective: there surely is an afterEach or similar to unset all of the data set up for the test suite in this file, might be a good idea to use it, not mandatory, but some of these variables, properties, objects, etc. can bleed out to other tests when running an entire battery of them, probably not an issue with Java, but might be worthy to check if needed.

Reply:
If I use @BeforeAll I will have that error and for sure I will use afterEach but with BeforeEach that thing does't happen